VERSION

```diff
@@ -1 +1 @@
-1.2.0
\ No newline at end of file
+1.2.1
```

Comment:
You forgot to bump the version in the library loader.
Next time don't perform these changes in a PR, I will handle versioning/releases.